src/scripting_api/util.js

```diff
@@ -20,11 +20,724 @@ class Util extends PDFObject {
     super(data);
 
     this._crackURL = data.crackURL;
+    this._scandCache = Object.create(null);
```

Comment:
Perhaps make this a `Map` instead, since that avoids e.g. using `in` to check if things exist in the cache?